numgrid/becke_partitioning.h

```diff
@@ -0,0 +1,14 @@
+#ifndef BECKE_PARTITIONING_H_INCLUDED
```

Comment:
Noted here, but valid for all header files. Use `#pragma once` instead of the guards. Avoids debugging missing `#endif` and the pain of finding unique names.

Reply:
Hm! Nice.